Project.toml

```diff
@@ -4,10 +4,13 @@ authors = ["Jérémie Gillet <jie.gillet@gmail.com> and contributors"]
 version = "1.3.1"
 
 [deps]
+ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
 ImageIO = "82e4d734-157c-48bb-816b-45c225c6df19"
 ImageMagick = "6218d12a-5da1-5696-b52f-db25d2ecc6d1"
+ImageTransformations = "02fcd773-0e25-5acc-982a-7f6622650795"
```

Comment:
ImageTransformations is a heavy dependency, I'd prefer to avoid it if possible. Where is this package used inside the `src/`?

Reply:
I use `imresize` to rescale images. The package is used in QRDecoders.jl too.
```julia
using Images
@edit imresize(rand(2,2), 2,2) # ImageTransformations/...
```

Reply:
Did I miss something? The diffs doesn't include anything contains the string "imresize".

Reply:
Sorry. You're right. It is used in test only.  The same mistake for ColorTypes.